relay_handler.js

```diff
@@ -421,6 +513,35 @@ LazyRelayOutReq.prototype.emitError =
 function emitError(err) {
     var self = this;
 
+    var now = self.channel.timers.now();
+    var elapsed = now - self.start;
+
+    self.channel.emitFastStat(self.channel.buildStat(
+        'tchannel.outbound.calls.per-attempt-latency',
+        'timing',
+        elapsed,
+        new stat.OutboundCallsPerAttemptLatencyTags(
+            self.inreq.serviceName,
+            self.inreq.callerName,
+            self.inreq.endpoint,
+            self.remoteAddr,
+            0
+        )
+    ));
+
+    self.channel.emitFastStat(self.channel.buildStat(
```

Comment:
It's not, this entire code path (`LazyRelay{In,Out}Req`) will never normally see an error frame error object, the two normal callers are:
- LazyRealyOutReq#onTimeout
- TChannelConnection#resetAll

The only other call is TChannelConnection#onCallErrorFrame, this could only happen during a transition time (when we flip off lazy handling e.g.).  I'd instead rather add a branch to this function that makes it call `req.handleLazily` if such function exists, or similar, thoughts?